src/cursor/cur_backup.c

```diff
@@ -272,6 +272,7 @@ __wt_curbackup_open(WT_SESSION_IMPL *session, const char *uri, WT_CURSOR *other,
       __wt_cursor_notsup,                             /* largest_key */
       __wt_cursor_notsup,                             /* cache */
       __wt_cursor_reopen_notsup,                      /* reopen */
+      __wt_cursor_checkpoint_id,                      /* checkpoint ID */
```

Comment:
What will other type of cursors got for this value? The latest checkpoint id? Then they will get different values from the same cursor if called multiple times?

Reply:
> What will other type of cursors got for this value? The latest checkpoint id? Then they will get different values from the same cursor if called multiple times?

As documented, they'll get either 0 (if they're not a checkpoint cursor), or the checkpoint ID of the checkpoint they were opened on. Calling them repeatedly will return the same value, the value is set on open and doesn't change.